CHANGES.md

```diff
@@ -1,3 +1,9 @@
+Unreleased
+=====
+
+* [#238](https://github.com/serokell/universum/issues/238):
```

Comment:
Btw, the link refers to `issues`, but 238 is this PR. Redirect works, but it looks weird :)

Reply:
Meh right, better be fixed